script/Arcueid/Day 6/06_00_ARC06_1_2.txt

```diff
@@ -324,12 +324,12 @@ She may be a powerful and beautiful woman, but a scowl like that would send a no
 [sha:c28dfe66a7af854820ccf14878c17a058b3f364d]{
 -- Page 106, Offset 14180.
 -- 「どうやら、兄さんには真面目に返答する気はないみたいですね」
-"It would appear that my brother has no intention of answering me seriously."
+"Sigh. It would appear that my brother has no intention of answering me seriously."
```

Comment:
Where's the sigh coming from here?

Reply:
akiha's voice

Reply:
Even so, not in the JP I'd say leave it out. I've never liked writing sigh in-text to begin with but I'm willing to just let it be when its in the original text.